ivy/functional/frontends/paddle/math.py

```diff
@@ -118,6 +118,12 @@ def atanh(x, name=None):
     return ivy.atanh(x)
 
 
+@with_supported_dtypes({"2.5.1 and below": ("int32", "int64")}, "paddle")
+@to_ivy_arrays_and_back
+def broadcast_shape(x, y):
```

Comment:
The args should be `x_shapes` and `y_shapes` according to the docs: https://www.paddlepaddle.org.cn/documentation/docs/en/2.2/api/paddle/broadcast_shape_en.html